api/constants/constants.go

```diff
@@ -91,6 +91,10 @@ const (
 
 	// DarwinOS is the GOOS constant for Apple macOS/darwin.
 	DarwinOS = "darwin"
+
+	// UseOfClosedNetworkConnection is a special string some parts of
+	// go standard lib are using that is the only way to identify some errors
+	UseOfClosedNetworkConnection = "use of closed network connection"
```

Comment:
use https://golang.org/pkg/net/#ErrClosed (assuming we stick with Go 1.16)

Reply:
This would require updating all places that rely on this error message to use `errors.Is` though, right? I wanted to avoid doing this in this PR as to not increase the scope too much. Also, I don't think we upgraded to 1.16 in v6 yet. I've left a todo item for now.

Reply:
Ahh, this constant was just moved from elsewhere, no problem